packages/data-models/src/inflection.js

```diff
@@ -214,28 +214,61 @@ class Inflection {
     return model.compareWords(wordA, wordB, normalize)
   }
 
+  /**
+   * Compare single feature values delegating to the language model
+   * rules for normalization
+   *
+   * @param {string} featureType the feature type
+   * @param {string} valueA the first value
+   * @param {string} valueB the secon value
+   * @param {boolean} normalize whether or not to apply normalization
+   */
+  modelCompareFeatureValue ( featureType, valueA, valueB, normalize = true )  {
+    const model = LMF.getLanguageModel(this.languageID)
+    return model.compareFeatureValue(featureType, valueA, valueB, normalize)
+  }
+
   /**
    * Check to see if the supplied inflection can disambiguate this one
    *
    * @param {Inflection} infl Inflection object to be used for disambiguation
+   * @param {Boolean} ignorePofs flag to ignore the inflection's part of speech
+   *                             (use if lexeme pofs is more relevant)
+   * @returns {Object} object { {Boolean} match, {Boolean} exactMatch }
+   *                   a match means the inflection was disamibugated
+   *                   an exactMatch means the disamibugator matched all
+   *                   values of all features
    */
-  disambiguatedBy (infl) {
+  disambiguatedBy (infl, ignorePofs = false) {
     let matched = true
+    let exactMatch = true
     // an inflection can only be disambiguated by its features
-    if (this.features.length === 0 || infl.features.length === 0) {
+    if (this.features.size === 0 || infl.features.size === 0) {
```

Comment:
this test actually wasn't working before, because `features` is a Set and doesn't have a `length` property.

Reply:
An excellent catch!